scripts/changelog-links.sh

```diff
@@ -10,7 +10,20 @@
 
 set -e
 
-SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
```

Comment:
`readlink -f` not supposed in BSD `readlink` so just changed strategies to expect pwd to always be root of repo